android/app/src/main/java/com/microsoft/codepush/react/CodePushUpdateManager.java

```diff
@@ -244,7 +245,16 @@ public void downloadPackage(JSONObject updatePackage, String expectedBundleFileN
                 }
 
                 if (isDiffUpdate) {
-                    CodePushUpdateUtils.verifyHashForDiffUpdate(newUpdateFolderPath, newUpdateHash);
+                    CodePushUtils.log("Applying diff update.");
```

Comment:
Don't see if we ever need this `if block` here due to we can move this `log` statement (and this is the only thing that this block is doing now) to Line 255 right after new `if (isDiffUpdate)` block begins.